### PR TITLE
CO-369

### PR DESCRIPTION
**Pull request by philippe89**

### CODE REVIEW

File: mailing_christmas/README.rst [hunk 1]
```diff
@@ -14,7 +14,6 @@ Use this module to send all the christmas mails.
 Contributors
 ------------
 
-* Emanuel Cino <ecino@compassion.ch>
```
Comment (ecino) [inline mailing_christmas/README.rst]: :sob:

File: mailing_christmas/report/payment_slip.py [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+from reportlab.pdfgen.canvas import Canvas
+from reportlab.lib.units import inch
+from openerp import models, api, _
+from openerp.tools.misc import mod10r
+
+FontMeta = namedtuple('FontMeta', ('name', 'size'))
+
+
+class PaymentSlip(models.Model):
+    """From Version 8 payment slip are now a
```
Comment (ecino) [inline mailing_christmas/report/payment_slip.py]: You can remove this comment and replace by "Add methods to print Christmas Payment Slip for partners"

File: mailing_christmas/report/report.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <report
+            id="abc"
```
Comment (ecino) [inline mailing_christmas/report/report.xml]: christmas_bvr_report would be better

**Comment by ecino** [comment]
Good job !

File: mailing_christmas/report/report.xml (hunk 3 above)
Comment (philippe89) [inline mailing_christmas/report/report.xml]: lol indeed :+1:

**Pull request merged**